doc/man/cockpit.conf.xml

```diff
@@ -84,6 +84,14 @@ Origins = https://somedomain1.com https://somedomain2.com:9090
              number of unauthenticated connections reaches <literal>full</literal> (60).</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><option>RedirectTLS</option></term>
```

Comment:
This name seems to focus on the implementation detail. How about 'AllowUnencrypted' or something similar?